daemon/cmd/kube_proxy_replacement.go

```diff
@@ -90,7 +90,7 @@ func initKubeProxyReplacementOptions() (strict bool) {
 		if option.Config.EnableIPSec {
 			msg := "IPSec cannot be used with BPF NodePort."
 			if strict {
-				log.Fatal(msg)
+				return false, fmt.Errorf("%v", msg)
```

Comment:
In some cases an error is formatted via `%v` and others aren't. We should just stick with one to be consistent. The `%v` seems redundant to me, so we can just do the latter.

Reply:
good catch, meant to align that. And I agree with your statement on how to do so.